### PR TITLE
Update: add rootstock

### PR DESCRIPTION
**Pull request by MustafaBrandon**

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/steer: 
        
 
  
    
    
    Error: 
        Unknown chain(s): rootstock
        Note: if you think that the chain is correct but missing from our list, please add it to 'projects/helper/chains.json' file
        
    Error saving cache: Error: ENOENT: no such file or directory, open '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
        at Object.openSync (node:fs:596:3)
        at Object.writeFileSync (node:fs:2322:35)
        at process.saveCache (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/cache.js:152:17)
        at process.emit (node:events:529:35)
        at process.exit (node:internal/process/per_thread:191:15)
        at process.handleError (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/utils/handleError.js:18:11)
        at process.emit (node:events:529:35)
        at emit (node:internal/process/promises:149:20)
        at processPromiseRejections (node:internal/process/promises:283:27)
        at process.processTicksAndRejections (node:internal/process/task_queues:96:32) {
      errno: -2,
      syscall: 'open',
      code: 'ENOENT',
      path: '/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@defillama/sdk/build/util/local_cache/zlib-1.0/BlockCache.json'
    }

File: projects/steer/index.js [hunk 1]
```diff
@@ -142,6 +142,13 @@ const supportedChains = [
     chainId: 196,
     identifier: 'xlayer'
   },
+  {
+    name: 'Rootstock',
+    subgraphEndpoint: 'https://api.goldsky.com/api/public/project_clohj3ta78ok12nzs5m8yag0b/subgraphs/steer-protocol-rootstock/1.1.1/gn',
+    chainId: 30,
+    identifier: 'rootstock'
```
Comment (g1nt0ki) [inline projects/steer/index.js:149]: change this to rsk

Comment (MustafaBrandon) [reply projects/steer/index.js]: bumped

**Comment by llamatester** [comment]
The adapter at projects/steer exports TVL: 
        
 
 
    linea                     4.34 M
    polygon                   3.21 M
    arbitrum-staking          3.06 M
    staking                   3.06 M
    metis                     2.05 M
    arbitrum                  1.78 M
    mode                      1.25 M
    manta                     1.05 M
    evmos                     873.21 k
    kava                      668.88 k
    base                      158.82 k
    scroll                    81.11 k
    telos                     77.28 k
    astar                     34.41 k
    bsc                       26.50 k
    fantom                    20.65 k
    astrzk                    9.19 k
    optimism                  5.22 k
    thundercore               434
    avax                      290
    polygon_zkevm             121
    blast                     90
    mantle                    11
    rsk                       4
    xlayer                    0
    
    total                    15.64 M

**Pull request merged**